src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryAuthenticationHandler.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Antiforgery.Internal;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features.Authentication;
+using Microsoft.AspNetCore.Http.Features.Authentication.Internal;
+
+namespace Microsoft.AspNetCore.Antiforgery.Internal
+{
+    public class AntiforgeryAuthenticationHandler : IAuthenticationHandler
+    {
+        public AntiforgeryAuthenticationHandler(IAntiforgery antiforgery)
+        {
+            Antiforgery = antiforgery;
+        }
+
+        protected IAntiforgery Antiforgery { get; }
```

Comment:
Similar to @Tratcher's [comment](https://github.com/aspnet/Antiforgery/pull/58/files#r53392610), `protected` seems weird in a `class` with no `virtual` methods.